packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/JoinInButton/JoinInButton.js

```diff
@@ -0,0 +1,26 @@
+import counterpart from 'counterpart'
+import { Button } from 'grommet'
+import Link from 'next/link'
+import { string } from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledButton = styled(Button)`
+  text-align: center;
+`
+
+function JoinInButton (props) {
+  const { href } = props
+  const label = counterpart('ZooniverseTalk.button')
+  return (
+    <Link href={href} passHref>
+      <StyledButton label={label} />
```

Comment:
Grommet's `Button` has an `href` prop that will render an anchor tag styled like a button. I think that's what the intention is here, so maybe you can set the `href` directly rather than wrap it with a `Link`?

Reply:
In this case we can, as we're directing the user towards Talk. Other anchors will require `Link` to enable client side routing - https://github.com/zeit/next.js/#dynamic-routing